zulip/integrations/google/google-calendar

```diff
@@ -166,7 +163,7 @@ def send_reminders() -> Optional[None]:
         message = "Reminder:\n\n" + "\n".join("* " + m for m in messages)
 
     result = zulip_client.send_message(
-        {"type": "direct", "to": [options.zulip_email], "content": message}
+        {"type": "direct", "to": [zulip_client.get_profile()["email"]], "content": message}
```

Comment:
The `zulip_client.get_profile()["email"]]` call should probably be at the beginning, not done every time one sends -- at least if we use a user ID rather than email. (I've not checked if later commits do that already).

Reply:
This moves around, but I think we might want to add a tweak to do `["id"]` instead.